src/app/(app)/forum/post/[postId]/page.tsx

```
@@ -0,0 +1,142 @@
+                
+                {/* Action Menu (3 dots) */}
+                <PostActionMenu post={post} isDetailView={true} />
+             </div>
+          </div>
+
+          {/* Content */}
+          <div className="p-6 space-y-6">
+             {/* Text Content */}
+             <div 
+               className="prose dark:prose-invert max-w-none text-foreground leading-relaxed"
+               dangerouslySetInnerHTML={{ __html: post.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)